dialogue-core/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     compile 'com.netflix.concurrency-limits:concurrency-limits-core'
     compile 'com.palantir.conjure.java.runtime:client-config'
     compile 'com.palantir.safe-logging:preconditions'
+    compile 'com.palantir.tracing:tracing'
```

Comment:
Can we use the `java-library` plugin and use an `implementation` dependency here?